kamon-play-2.4.x/src/main/scala/kamon/play/instrumentation/WSInstrumentation.scala

```diff
@@ -34,45 +34,42 @@ class WSInstrumentation {
 
   @Around("onExecuteRequest(request)")
```

Comment:
would be too much to ask to change this to also instrument `WSClient.url()` method?

Reply:
I wanted to but turns out there are no request filters in 2.4... the most I can do is hook on `play.api.libs.ws.WSRequest.execute()` to keep it general.